index.html

```diff
@@ -22,7 +32,22 @@ <h2 class="project-title">Add a new book</h2>
         <button class="add-btn" type="submit">Add</button>
       </form>
     </section>
+    <section id="contact">
+      <h2>Contact information</h2>
+      <p>Do you have any questions or you just want to say "Hello"?</p>
+      <ul>
+        <li>Our e-mail: africadev@mail.com</li>
+        <li>Our phone number: 001122334455</li>
+        <li>Our address: The moon</li>
+      </ul>
+    </section>
+
+    <footer>
+      Copyright © AFRICADev 2021
```

Comment:
I am really impressed with the work and effort you have put to make UI amazing. 
Please follow the wireframe for the footer. Footer text should be on the left side of the page.